src/filemanager.cpp

```diff
@@ -1489,7 +1489,7 @@ static void *pj_open_lib_internal(
                       .empty()) {
             auto paths = NS_PROJ::internal::split(projLib, dirSeparator);
             for (const auto &path : paths) {
-                fname = path;
+                fname = NS_PROJ::internal::stripQuotes(path);
```

Comment:
should be done also on line 1524. We might use a lamba for lines 1490 et 1499 that could be use in both places

Reply:
I did not look for other uses of `projLib`. Good point. I've updated the PR.